src/Wallabag/CoreBundle/Tests/Controller/EntryControllerTest.php

```diff
@@ -31,6 +31,30 @@ public function testGetNew()
         $this->assertCount(1, $crawler->filter('button[type=submit]'));
     }
 
+    public function testPostNewViaBookmarklet()
+    {
+        $this->logInAs('admin');
+        $client = $this->getClient();
+
+        $crawler = $client->request('GET', '/');
+
+        $this->assertCount(4, $crawler->filter('div[class=entry]'));
+
+        // Good URL
+        $crawler = $client->request('GET', '/bookmarklet', array('url' => $this->url));
+        $this->assertEquals(302, $client->getResponse()->getStatusCode());
+        $crawler = $client->followRedirect();
+        $crawler = $client->request('GET', '/');
+        $this->assertCount(5, $crawler->filter('div[class=entry]'));
+
+        // Bad URL
+        $crawler = $client->request('GET', '/bookmarklet', array('url' => 'fakeURL'));
+        $this->assertEquals(302, $client->getResponse()->getStatusCode());
+        $crawler = $client->followRedirect();
+        $crawler = $client->request('GET', '/');
+        $this->assertCount(5, $crawler->filter('div[class=entry]'));
+    }
+
```

Comment:
Instead of updating all count below, you should better remove the entry after added it.
You can remove it using the url.